docs/quarkifier.md

```
@@ -7,8 +7,17 @@ The Quarkifier (`com.clementguillot.quarkifier`) is a standalone Java tool that
 
 ## CLI Interface
 
+The top-level command dispatches to `augmentation` and `enrich-extension`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)